### ISSUE TITLE
Clicking remove employee with no employee selected removes all employees from GUI and causes further errors

### ISSUE THREAD

**Issue by smeth194243** [Locke087/EmpData#3]
I tried to remove an employee, but didn't click on them

Screen looks like this now
![image](https://user-images.githubusercontent.com/21316661/205344499-a0fdf1d5-5d50-45c2-b382-6234f2f5de94.png)

All of the employees still have information in the database but not in the GUI

I tried restarting the program

Error(s) upon running the program and logging in: 

cd : Cannot find path 'C:\Users\School\OneDrive - Utah Valley University\Desktop\Repositories\EmpData\EmpData' because it     
does not exist.
At C:\Users\School\OneDrive - Utah Valley University\Desktop\Repositories\EmpData\run.ps1:1 char:1
+ cd EmpData
+ ~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\School...EmpData\EmpData:String) [Set-Location], ItemNotFoundExcepti  
   on
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.SetLocationCommand
 
Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Python310\lib\tkinter\__init__.py", line 1921, in __call__
    return self.func(*args)
  File "C:\Users\School\OneDrive - Utah Valley University\Desktop\Repositories\EmpData\login.py", line 70, in login
    self.app = ViewEmployeeAdmin(self.master, employee)
  File "C:\Users\School\OneDrive - Utah Valley University\Desktop\Repositories\EmpData\view_employees.py", line 66, in __init__
    if employee.is_deactivated == 'n':
AttributeError: 'Employee' object has no attribute 'is_deactivated'

Once logged in, user can add an employee, and their information is added to the database, but not shown in the GUI

New rows in employeetest.csv:
1234,Admin,Example,Executive,Making fat stacks,lovemytesla@gmail.com,1234 Address dr,,City,State,12345,Country,123-456-7891,Salary,111111111,11/11/2022,111-11-1111,12/1/2022,,11111111111,admin,wife,n,5e884898da28047151d0e56f8dc6292773603d0d6aabbdd62a11ef721d1542d8

1234,Admin,Example,Executive,Making fat stacks,lovemytesla@gmail.com,123 Address dr,,city,state,12345,country,123-456-7891,Salary,111111111,111/11/2022,111-11-1111,12/1/2022,,1111111111,admin,wife,n,5e884898da28047151d0e56f8dc6292773603d0d6aabbdd62a11ef721d1542d8

123456,**Database,Workshopefully**,dep,title,email@service.com,12345 address blvd,,city,state,98745,country,987-654-3521,Hourly,2,5/23/2010,987-65-4321,12/12/2021,,987654321,admin,wife,n,e7cf3ef4f17c3999a94f2c6f612e8a888e5b1026878e4e19398b23bd38ec221a

Basically removing all the employees from the GUI wrecks everything, and the user shouldn't be able to do that in the first place.

**Comment by kodecreer** [comment]
Did you insert that data into the csv or did you do it through the GUI?

**Comment by smeth194243** [comment]
Through GUI

**Comment by smeth194243** [comment]
Can't replicate anymore.

**Comment by smeth194243** [comment]
@kodecreer I got it to break again

remove an employee
click view employee
back button
